### ISSUE TITLE
Feature Request: Add Support for Ubuntu x86

### ISSUE THREAD

**Issue by houlem** [qonto/prometheus-rds-exporter#25]
Thanks for being active in this project.

What: To have the ability to install just the rds exporter on Debian x86.

In my current setup I have Prometheus federated instances already setup and running I just want to add the exporter to my ec2 instance that’s I have already running.

**Comment by vmercierfr** [comment]
As we talked about, we'll add Debian package (Ubuntu compatible) to the release cycle next week.

Meanwhile, the workaround is to start it manually using binary:

```bash
# IAM permissions requirement
# - The EC2 instance must have the mentioned in https://github.com/qonto/prometheus-rds-exporter#aws-authentication
# - or valid credentials in shell with "aws configure"
 
# Download exporter
PROMETHEUS_RDS_EXPORTER_VERSION=0.2.8
wget https://github.com/qonto/prometheus-rds-exporter/releases/download/${PROMETHEUS_RDS_EXPORTER_VERSION}/prometheus-rds-exporter_Linux_$(uname -m).tar.gz
tar xvzf prometheus-rds-exporter_Linux_x86_64.tar.gz

# Start the exporter
export AWS_DEFAULT_REGION=eu-west-3 # replace with your AWS region
./prometheus-rds-exporter
```

**Comment by vmercierfr** [comment]
Hello @houlem,

We developed the code to release Debian packages and updated instructions in the README. It should be live on Wednesday in 0.3.0.

I keep you in touch as soon as it's available.

**Comment by houlem** [comment]
Amazing thank you so much! I can test it out on my end then.

**Comment by vmercierfr** [comment]
We just released the [v0.3.0](https://github.com/qonto/prometheus-rds-exporter/releases/tag/0.3.0). Now we are publishing Debian packages that are compatible with Debian and Ubuntu.

For convenience, we also implemented automatic AWS region detection using AWS EC2 IMDS, so you don't need environment variables anymore. It should work out-of-the-box (if the EC2 instance has the [required IAM policies](https://github.com/qonto/prometheus-rds-exporter#aws-authentication))

We documented all the steps to deploy on the AWS EC2 instance in the [README](https://github.com/qonto/prometheus-rds-exporter#debianubuntu-on-aws-ec2).
